demos/python_demos/image_retrieval_demo/image_retrieval_demo.py

```diff
@@ -137,14 +139,15 @@ def main():
             else:
                 log.info("ROI detected, found: %s", sorted_classes[0])
 
-        key = visualize(view_frame, position,
-                        [img_retrieval.impaths[i] for i in sorted_indexes],
-                        distances[sorted_indexes] if position is not None else None,
-                        img_retrieval.input_size, np.mean(compute_embeddings_times),
-                        np.mean(search_in_gallery_times), imshow_delay=3)
+        if not args.no_show:
+            key = visualize(view_frame, position,
```

Comment:
I suggest forwarding that key to `visualize()`. We want to disable only `imshow` and `waitkey` while this approach disables the whole function which we could test.

Reply:
Ok